letsdrill.rb

```diff
@@ -1,12 +1,26 @@
 def get_letter_grade(integer)
  
-  #Put your code here!
+  if integer >= 90
+  	return "A"
+  elsif integer >= 80
+  	return "B"
+  elsif integer >= 70
+  	return "C"
+  elsif integer >= 60
+  	return "D"
+  else
+  	return "F"
+  end
 
 end
 
 def shortest_string(array)
 
-  #Put your code here!
+  if array.empty?
+  	return nil
+  end
+
+  return array.sort_by {|x| x.length}[0]
```

Comment:
This will work.
Consider #first instead of [0].  Or better yet, #min_by instead of #sort_by.